fluent: admin: update supported platforms for style (#1492)
fluent/fluent-bit-docs: installation/supported-platforms.md

|  | [Rocky Linux 8](linux/redhat-centos.md) | x86_64, Arm64v8 |
|  | [Alma Linux 8](linux/redhat-centos.md) | x86_64, Arm64v8 |
|  | [Debian 12 (Bookworm)](linux/debian.md) | x86_64, Arm64v8 |
|  | [Debian 11 (Bullseye)](linux/debian.md) | x86_64, Arm64v8 |
|  | [Debian 10 (Buster)](linux/debian.md) | x86_64, Arm64v8 |
|  | [Ubuntu 24.04 (Noble Numbat)](linux/ubuntu.md) | x86_64, Arm64v8 |
|  | [Ubuntu 22.04 (Jammy Jellyfish)](linux/ubuntu.md) | x86_64, Arm64v8 |
|  | [Ubuntu 20.04 (Focal Fossa)](linux/ubuntu.md) | x86_64, Arm64v8 |
|  | [Ubuntu 18.04 (Bionic Beaver)](linux/ubuntu.md) | x86_64, Arm64v8 |
|  | [Ubuntu 16.04 (Xenial Xerus)](linux/ubuntu.md) | x86_64 |
|  | [Raspbian 11 (Bullseye)](linux/raspbian-raspberry-pi.md) | Arm32v7 |
|  | [Raspbian 10 (Buster)](linux/raspbian-raspberry-pi.md) | Arm32v7 |
| macOS | * | x86_64, Apple M1 |
| Windows | [Windows Server 2019](windows.md) | x86_64, x86 |
|  | [Windows 10 1903](windows.md) | x86_64, x86 |

From an architecture support perspective, Fluent Bit is fully functional on x86_64,
Arm64v8 and Arm32v7 based processors.

Fluent Bit can work also on OSX and BSD systems, but not all plugins will be
available on all platforms.

> Official support will be expanding based on community demand. Fluent Bit might run on
> older operating systems, but must be built from source, or using custom packages from

Reviewer: ```suggestion
Official support is based on community demand. Fluent Bit might run on older operating
systems, but must be built from source, or using custom packages from
```
